src/colony_print/controllers/node.py

```diff
@@ -86,14 +87,31 @@ def job_result(self, id, job_id):
     @appier.route("/nodes/<str:id>/print", ("GET", "POST"), json=True)
     @appier.ensure(token="admin")
     def print_default(self, id):
-        data_b64 = self.field("data_b64", mandatory=True, not_empty=True)
+        data = self.field("data", None)
+        data_b64 = self.field("data_b64", None)
         name = self.field("name", None)
         type = self.field("type", None)
         format = self.field("format", None)
         options = self.field("options", None)
 
+        appier.verify(
+            data or data_b64,
+            message="Either data or data_b64 fields must be provided",
+            code=400,
+        )
+        appier.verify(
+            not (data and data_b64),
+            message="Only one of data or data_b64 fields must be provided",
+            code=400,
+        )
+
         job_id = str(uuid.uuid4())
         name = name or job_id
+        data_b64 = (

```

Comment:
```python
if data:
  data_b64 = base64.b64encode(appier.legacy.bytes(data, encoding="utf-8")).decode('utf-8')
```

would argue that this style is more straightforward 😅

Reply:
That is probably right b64encode returns a bytes object and we should offer a str

Reply:
But we can use appier.legacy.str for that i guess

Reply:
Now thinking about it, I believe your option is better :)

Reply:
Done https://github.com/hivesolutions/colony-print/commit/669988376f3f285067fb515e758e5f4ec7393c26